server.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express');
+const db = require('./data/db')
+
+const port = 5555;
+const server = express();
+server.use(express.json());
+
+server.get('/', (req, res) => {
```

Comment:
Good Boilerplate, Consider removing this get endpoint when finished, since it is not a required part of the API.